ingestion/src/metadata/cli/db_dump.py

```diff
@@ -114,10 +114,10 @@ def get_hash_column_name(engine: Engine, table_name: str) -> Optional[str]:
     inspector = inspect(engine)
     columns = inspector.get_columns(table_name)
     for column in columns:
-        if column["name"] == FQN_HASH_COLUMN:
-            return FQN_HASH_COLUMN
-        if column["name"] == NAME_HASH_COLUMN:
-            return NAME_HASH_COLUMN
+        if column["name"].lower() == FQN_HASH_COLUMN.lower():
```

Comment:
what are we doing here, any reason to compare the name to fan_hash

Reply:
we only [select](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/cli/db_dump.py#L40) a few columns for backup.
We have to check the table which column name it has (fqnHash or nameName). The condition here returns that name